.github/workflows/greetings.yml

```diff
@@ -0,0 +1,13 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Message that will be displayed on users'' first issue'
+        pr-message: 'Message that will be displayed on users'' first pr'
```

Comment:
We should update these messages to what we actually want them to say :) Would be good to find some other examples maybe

Reply:
Gotcha - will do some more research. And yeah - totally forgot to change those, ha-ha, thanks!

Reply:
I wonder if we already have some community standards for greetings - @binford2k ?

Reply:
We don't yet. There are some projects using https://probot.github.io/apps/welcome/ right now.

I've been waiting until the bot supports org-wide first contributors before standardizing on something. Can this action support org-wide? If so, then we could add it to template repos intended for public projects.

Reply:
Ladies and gentlemen, should I try to come up with something or it's better an epic in Jira to be opened and start off from there?